csm/cli/support_bundle/csm_bundle_generate.py

```diff
@@ -111,16 +113,58 @@ class GenerateCsmBundle:
     '''
     @staticmethod
     def generate_bundle(args):
+
         Conf.load(const.CONSUMER_INDEX, args['config_url'])
         log_path = Conf.get(const.CONSUMER_INDEX, const.CSM_LOG_PATH_KEY)
-        csm_log_path = os.path.join(log_path,'csm')
+        csm_log_path = os.path.join(log_path, const.CSM_COMPONENT_NAME)
         bundle_id = args['bundle_id']
         target_path = args ['target']
-        services = args['services'] #Not making any use of service option for now
-        target_path = os.path.join(target_path, bundle_id)
+        duration = args['duration']
+        size_limit = args['size_limit']
+        # services = args['services'] #Not making any use of service option for now
```

Comment:
Remove the comment

Reply:
By removing the comments coday is failing for unused vars. So as u suggested initialized all the vars at class level.
Fixed.